src/styles/blocks/fonts.scss

```diff
@@ -0,0 +1 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
```

Comment:
please check how to add google fonts properly to your project (read [here](https://www.w3schools.com/css/css_font_google.asp))